[2022-08-06 19:38:01,187] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 19:38:01,204] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 19:38:01,204] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:01,204] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:01,204] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:01,232] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-07 00:00:00+00:00
[2022-08-06 19:38:01,240] {standard_task_runner.py:52} INFO - Started process 1925 to run task
[2022-08-06 19:38:01,246] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '587', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpr5h38evc', '--error-file', '/tmp/tmphmk88td6']
[2022-08-06 19:38:01,249] {standard_task_runner.py:80} INFO - Job 587: Subtask second_task
[2022-08-06 19:38:01,355] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:01,478] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-08-06 19:38:01,479] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:01,480] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:01,502] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:01,505] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:01,506] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:01,557] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220207T000000, start_date=20220806T193801, end_date=20220806T193801
[2022-08-06 19:38:01,617] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:01,665] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:52,223] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 21:51:52,237] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 21:51:52,237] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:52,237] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:52,237] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:52,255] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-07 00:00:00+00:00
[2022-08-06 21:51:52,261] {standard_task_runner.py:52} INFO - Started process 2109 to run task
[2022-08-06 21:51:52,264] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '582', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpccugenvw', '--error-file', '/tmp/tmp0bzyzzp7']
[2022-08-06 21:51:52,264] {standard_task_runner.py:80} INFO - Job 582: Subtask second_task
[2022-08-06 21:51:52,336] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:52,422] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-08-06 21:51:52,423] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:52,423] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:52,437] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:52,439] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:52,439] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:52,468] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220207T000000, start_date=20220806T215152, end_date=20220806T215152
[2022-08-06 21:51:52,515] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:52,549] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:15,403] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 21:57:15,416] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 21:57:15,416] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:15,416] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:15,416] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:15,431] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-07 00:00:00+00:00
[2022-08-06 21:57:15,436] {standard_task_runner.py:52} INFO - Started process 1895 to run task
[2022-08-06 21:57:15,439] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '581', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4e2aesi8', '--error-file', '/tmp/tmp34325wnj']
[2022-08-06 21:57:15,439] {standard_task_runner.py:80} INFO - Job 581: Subtask second_task
[2022-08-06 21:57:15,535] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:15,607] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-08-06 21:57:15,608] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:15,609] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:15,619] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:15,621] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:15,621] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:15,648] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220207T000000, start_date=20220806T215715, end_date=20220806T215715
[2022-08-06 21:57:15,690] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:15,718] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:19,202] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 22:12:19,219] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 22:12:19,219] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:19,219] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:19,219] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:19,241] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-07 00:00:00+00:00
[2022-08-06 22:12:19,249] {standard_task_runner.py:52} INFO - Started process 1917 to run task
[2022-08-06 22:12:19,252] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '583', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpkx6ywa09', '--error-file', '/tmp/tmpr4j67kr1']
[2022-08-06 22:12:19,253] {standard_task_runner.py:80} INFO - Job 583: Subtask second_task
[2022-08-06 22:12:19,325] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:19,409] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-08-06 22:12:19,410] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:19,410] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:19,424] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:19,425] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:19,426] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:19,465] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220207T000000, start_date=20220806T221219, end_date=20220806T221219
[2022-08-06 22:12:19,507] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:19,546] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:39,226] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 22:23:39,240] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-08-06 22:23:39,240] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:39,240] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:39,240] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:39,265] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-07 00:00:00+00:00
[2022-08-06 22:23:39,271] {standard_task_runner.py:52} INFO - Started process 1961 to run task
[2022-08-06 22:23:39,274] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '585', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp82vmlcbc', '--error-file', '/tmp/tmp10rcvxic']
[2022-08-06 22:23:39,274] {standard_task_runner.py:80} INFO - Job 585: Subtask second_task
[2022-08-06 22:23:39,341] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:39,428] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-08-06 22:23:39,429] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:39,429] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:39,443] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:39,444] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:39,445] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:39,485] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220207T000000, start_date=20220806T222339, end_date=20220806T222339
[2022-08-06 22:23:39,526] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:39,557] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
